README.md

```diff
@@ -1,15 +1,96 @@
-# json5 loader for webpack
+[![npm][npm]][npm-url]
+[![deps][deps]][deps-url]
+[![test][test]][test-url]
+[![coverage][cover]][cover-url]
+[![chat][chat]][chat-url]
 
-## Usage
+<div align="center">
+  <!-- replace with accurate logo e.g from https://worldvectorlogo.com/ -->
+  <img width="280" height="200" style=""
+    src="https://cdn.rawgit.com/json5/json5-logo/master/json5-logo.svg">
+  <a href="https://github.com/webpack/webpack">
+    <img width="200" height="200" vspace="" hspace="25"
+      src="https://worldvectorlogo.com/logos/webpack.svg">
+  </a>
+  <h1>JSON5 Loader</h1>
+  <p>Webpack loader for parsing <code>json5</code> files into JavaScript objects.<p>
+</div>
 
-``` javascript
+<h2 align="center">Install</h2>
+
+```sh
+$ npm i -D json5-loader
+```
+
+<h2 align="center">Usage</h2>
+
+```js
 var json = require("json5!./file.json5");
```

Comment:
Could we add the more common use-case with the webpack config? I think having this weird stuff in `require` is really off-putting to people.